grug/testing/src/suite.rs

```diff
@@ -629,10 +629,10 @@ where
     ID: Indexer,
     AppError: From<DB::Error> + From<VM::Error> + From<PP::Error> + From<ID::Error>,
 {
-    type Error = AppError;
-
-    fn query_chain(&self, req: Query) -> AppResult<QueryResponse> {
-        self.app.do_query_app(req, 0, false)
+    fn query_chain(&self, req: Query) -> StdResult<QueryResponse> {
+        self.app
+            .do_query_app(req, 0, false)
+            .map_err(|err| StdError::host(err.to_string()))
```

Comment:
`TestSuite` is the only `Querier` implementation of which the error type isn't `StdError`. Here we simply cast it to `StdError::Host`